boards/DCM/Src/App/states/App_InitState.c

```diff
@@ -15,20 +16,71 @@ static void InitStateRunOnEntry(struct StateMachine *const state_machine)
 
 static void InitStateRunOnTick1Hz(struct StateMachine *const state_machine)
 {
-    App_AllStatesRunOnTick1Hz(state_machine);
+    App_SharedStatesRunOnTick1Hz(state_machine);
 }
 
 static void InitStateRunOnTick100Hz(struct StateMachine *const state_machine)
 {
-    App_AllStatesRunOnTick100Hz(state_machine);
+    App_SharedStatesRunOnTick100Hz(state_machine);
+
+    // Holds previous start switch position (true = UP/ON, false = DOWN/OFF)
+    static bool prev_start_switch_position =
```

Comment:
thoughts on renaming this to:
```suggestion
    static bool is_start_switch_prev_upwards =
```